docs/nidcpower/rep_caps.rst

```diff
@@ -30,10 +33,15 @@ channels
 
         .. code:: python
 
-            session.channels['0-2'].channel_enabled = True
+            session.channels[0].output_function = nidcpower.OutputFunction.DC_CURRENT
+
+        sets :py:attr:`output_function` to :py:data:`~nidcpower.OutputFunction.DC_CURRENT` for channels 0.
+
+        .. code:: python
 
-        passes a string of :python:`'0, 1, 2'` to the set attribute function.
+            session.channels[0, 2].output_function = nidcpower.OutputFunction.DC_CURRENT
 
```

Comment:
here is where we could add an example using an iterable, below line 42?

session.channles[range(3)].output_function = nidcpower.OutputFunction.DC_CURRENT